tests/FSharp.Compiler.ComponentTests/EmittedIL/Inlining/Match01_RealInternalSignatureOn.fs.il.net472.debug.bsl

```diff
@@ -1003,48 +783,168 @@
       IL_000c:  ret
     } 
 
-    .method public hidebysig virtual final 
-            instance int32  CompareTo(object obj,
-                                      class [runtime]System.Collections.IComparer comp) cil managed
+    .method assembly static int32  'CompareTo$cont@4-1'(class Match01/Test1 this,
+                                                        class Match01/Test1 objTemp,
+                                                        class [FSharp.Core]Microsoft.FSharp.Core.Unit unitVar) cil managed
     {
       .custom instance void [runtime]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = ( 01 00 00 00 ) 
       
-      .maxstack  4
-      .locals init (class Match01/Test1 V_0,
-               class [FSharp.Core]Microsoft.FSharp.Core.FSharpFunc`2<class [FSharp.Core]Microsoft.FSharp.Core.Unit,int32> V_1)
+      .maxstack  5
```

Comment:
Seeing the amount of locals below, is the .maxstack 5 correct here?

Reply:
I think the merge of main fixed this. there is no longer a change in this baseline